Rawl_B502/alia_peritia_de_genelogiis_&_de_generibus_hibernensium.ttl

```diff
@@ -880,12 +880,12 @@ _:missing-2d466a7a
 <#ConallCernach>
     a foaf:Person ;
     irishRel:nomName "Conall Cernach" ;
-    owl:sameAs <http://example.com/LL/laigsi.ttl#ConaillCernaig>, <http://example.com/LL/dimittamus_in_terim.ttl#ConaillCernaig>, <http://example.com/LL/dail_araide.ttl#ConaillCernaig>, <http://example.com/LL/dáil_araide.ttl#ConaillCernaig>, <http://example.com/LL/lagin.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/do_forslointib_ulad_iar_coitchiund_in_so.ttl#ConallCernach>, <http://example.com/Rawl_B502/genelach_úa_n_echach.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/na_ríg_immorro_iar_conchobor.ttl#ConaillCernaich>.
+    owl:sameAs <http://example.com/LL/laigsi.ttl#ConaillCernaig>, <http://example.com/LL/dimittamus_in_terim.ttl#ConaillCernaig>, <http://example.com/LL/dail_araide.ttl#ConaillCernaig>, <http://example.com/LL/dáil_araide.ttl#ConaillCernaig>, <http://example.com/LL/lagin.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/do_forslointib_ulad_iar_coitchiund_in_so.ttl#ConallCernach>, <http://example.com/Rawl_B502/genelach_úa_n_echach.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/na_ríg_immorro_iar_conchobor.ttl#ConaillCernaich>;
```

Comment:
I presume this is the solution to Conall Cernach's other occurrences not displaying in the visualization. This all seems valid but we should discuss how it is to be done in future and whether it will be possible to retrieve the sameAs links for individuals with less distinctive names. In other words, did you retrieve them using reasoning or via a search for the name string?

Reply:
I made them by string search and visual inspection.  Yes, we do need to discuss this and I think it would be best in email.